public/components/Flyout/flyouts/components/AlertsDashboardFlyoutComponent.js

```diff
@@ -0,0 +1,586 @@
+  };
+
+  getBucketLevelGraphFilter = (trigger) => {
+    const compositeAggFilter = _.get(trigger, 'condition.composite_agg_filter');
+    if (_.isEmpty(compositeAggFilter)) return DEFAULT_WHERE_EXPRESSION_TEXT;
+    const keyword = _.keys(compositeAggFilter)[0];
+    const operator = _.keys(compositeAggFilter[keyword])[0];
+    const value = _.get(compositeAggFilter, `${keyword}.${operator}`);
+    return `${keyword} ${_.upperCase(operator)}S ${value}`;
+  };
+
+  getAlerts = async () => {
+    this.setState({ ...this.state, loading: true });
+    const {
+      from,
+      search,
+      sortField,
+      sortDirection,
+      severityLevel,
+      alertState,
+      monitorIds,
+    } = this.state;
+
+    const { httpClient, history, notifications, triggerID } = this.props;
+
+    const params = {
+      from,
+      size: MAX_ALERT_COUNT,
+      search,
+      sortField,
+      sortDirection,
+      severityLevel,
+      alertState,
+      monitorIds,
+    };
+
+    const queryParamsString = queryString.stringify(params);
+    history.replace({ ...this.props.location, search: queryParamsString });
+
+    httpClient.get('../api/alerting/alerts', { query: params }).then((resp) => {
+      if (resp.ok) {
+        const { alerts } = resp;
+        const filteredAlerts = _.filter(alerts, { trigger_id: triggerID });
+        this.setState({
+          ...this.state,
+          alerts: filteredAlerts,
+          totalAlerts: filteredAlerts.length,
+        });
+      } else {
+        console.log('error getting alerts:', resp);
+        backendErrorNotification(notifications, 'get', 'alerts', resp.err);
+      }
+    });
+
+    this.setState({ ...this.state, loading: false });
+  };
+
+  acknowledgeAlerts = async () => {
+    const { selectedItems } = this.state;
+    const { httpClient, notifications } = this.props;
+
+    if (!selectedItems.length) return;
+
+    const selectedAlerts = filterActiveAlerts(selectedItems);
+
+    const monitorAlerts = selectedAlerts.reduce((monitorAlerts, alert) => {
+      const { id, monitor_id: monitorId } = alert;
+      if (monitorAlerts[monitorId]) monitorAlerts[monitorId].push(id);
+      else monitorAlerts[monitorId] = [id];
+      return monitorAlerts;
+    }, {});
+
+    const promises = Object.entries(monitorAlerts).map(([monitorId, alerts]) =>
+      httpClient
+        .post(`../api/alerting/monitors/${monitorId}/_acknowledge/alerts`, {
+          body: JSON.stringify({ alerts }),
+        })
+        .then((resp) => {
+          if (!resp.ok) {
+            backendErrorNotification(notifications, 'acknowledge', 'alert', resp.resp);
+          } else {
+            const successfulCount = _.get(resp, 'resp.success', []).length;
+            displayAcknowledgedAlertsToast(notifications, successfulCount);
+          }
+        })
+        .catch((error) => error)
+    );
+
+    const values = await Promise.all(promises);
+    console.log('values:', values);
```

Comment:
Can remove this if it was used for testing

Reply:
This is a log that's present in the `Dashboard.js` `acknowledgeAlert` [implementation](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/public/pages/Dashboard/containers/Dashboard.js#L285). It seemed worthwhile to maintain it in this similar function.

Reply:
Personally, I'd say it should be removed from there as well. We typically use `console.log` for debugging purposes.

Reply:
Makes sense, I'll remove both instances.